yew-macro/src/html_tree/html_dashed_name.rs

```diff
@@ -9,7 +9,7 @@ use syn::ext::IdentExt;
 use syn::parse::{Parse, ParseStream, Result as ParseResult};
 use syn::Token;
 
-#[derive(PartialEq)]
+#[derive(Clone, PartialEq)]
```

Comment:
I added this so it's possible to create a `TagKey` from a `TagName` further down. This isn't strictly required as `TagKey` could just use the string representation instead but I felt it made more sense to do it this way

Reply:
Totally fine!